app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ActiveRecord::Base
+ has_many :lists, :dependent => :delete_all
+ has_many :items, :dependent => :delete_all
```

Comment:
It doesn't look like a user has_many items.  This would require a `user_id` column to be on `Items`, and probably is not what you are looking for.. since you can get to items by going through lists.

You can also see this doesn't work by doing something `User.first.items`